common/environments/deepspeed-mii-deployment/context/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM nvidia/cuda:11.3.1-devel-ubuntu18.04:{{latest-image-tag}}
```

Comment:
we have cuda 11.3 image that is regularly patched, please consider to use that or greater version of cuda. if you still want to use the one from nvidia, I would recommend to use the one from nvcr, dockerhub images are not supported well (unless you want to take care of every single vulnerability manually)